storage/test/conftest.py

```diff
@@ -1,6 +1,23 @@
 import pytest
 
+from storage.kv_in_memory import KeyValueStorageInMemory
+from storage.kv_store import KeyValueStorage
+from storage.kv_store_leveldb import KeyValueStorageLeveldb
+
 
 @pytest.fixture(scope='function')
 def tempdir(tmpdir_factory):
     return tmpdir_factory.mktemp('').strpath
+
+
+@pytest.yield_fixture(params=['memory', 'leveldb'])
+def parametrised_storage(request, tmpdir_factory) -> KeyValueStorage:
```

Comment:
Why don't we test it with FileStorage?

Reply:
`FileStorage` should not be used for key value storage, it is not efficient.